docs/content/developer-guide/ark-sdk.mdx

```diff
@@ -64,6 +64,21 @@ npm run generate:api
 # lib/api/generated/types.ts
 ```
 
+## Adding Custom SDK Functionality
+
+The ARK SDK supports overlay files for adding custom functionality beyond generated types:
```

Comment:
simplify this, no bold, briefly explain what part of the sdk is generated and what overlay is